mkt/extensions/models.py

```diff
@@ -160,19 +159,33 @@ def is_public(self):
         return self.status == STATUS_PUBLIC
 
     @property
-    def manifest_url(self):
-        return absolutify(reverse('extension.manifest',
+    def mini_manifest_url(self):
+        return absolutify(reverse('extension.mini_manifest',
                                   kwargs={'uuid': self.uuid}))
 
     @property
     def mini_manifest(self):
-        """Mini-manifest used for install/update in dict form.
-
-        Actually a maxi-manifest for the moment. Must contain `package_path`,
-        that points to the extension download absolute URL."""
-
-        mini_manifest = deepcopy(self.manifest)
-        mini_manifest['package_path'] = self.download_url
+        """Mini-manifest used for install/update on FxOS devices, in dict form.
+
+        It follows the Mozilla App Manifest format, *not* the Web Extension
+        manifest format.
```

Comment:
Regarding the why:

Something like ```It follows the Mozilla App Manifest format (because that's what Firefox OS requires to install/update add-ons), *not* the Web Extension manifest format.``` ?

It was not obvious to me at all either. It happens because internally, add-ons are actually implemented as Webapps, they are listed in `mozApps.getInstalled()` etc. I'm not sure how much we want to elaborate on that in the docstring.

How:

It's described below in the comment. I felt it was too internal for it to be part of the docstring.